src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__container">
+        <div class="calendar__month calendar__month--month--length--29 calendar__month--first--day--sun">
+          <div class="calendar__day calendar__day--day--1"></div>
```

Comment:
I believe that you could complete this task without this modifiers for every element 🙂